wiki/Music_theory/Rhythm/en.md

```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+---
+
+# Rhythm
+
+**Rhythm** is the pattern of sound, silence, and emphasis in a song.[[1]][r] In osu, mappers utilise these rythms when creating beatmaps, notes are represented by [beats](/wiki/Music_theory/Beat) and accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound).
```

Comment:
other than that, it kind of works as a stub

```suggestion
**Rhythm** is the pattern of sound, silence, and emphasis in a song.[[1]][r] In osu!, mappers utilise these combinations when creating beatmaps; notes are represented by [beats](/wiki/Music_theory/Beat) and accents are represented by [hitsounds](/wiki/Beatmapping/Hitsound).
```

Reply:
weren't we removing the exclamation or did i miss something?

Reply:
what? no, it's the branding